src/languages/en.ts

```diff
@@ -1833,6 +1833,28 @@ export default {
     session: {
         offlineMessageRetry: "Looks like you're offline. Please check your connection and try again.",
     },
+    travel: {
+        header: 'Book travel',
+        title: 'Travel smart',
+        subtitle: 'Use Expensify Travel to get the best travel offers and manage all your business expenses in one place.',
+        features: {
+            saveMoney: 'Save money on your bookings',
+            alerts: 'Get real time updates and alerts',
```

Comment:
This should be "realtime" instead of "real time". cc @jamesdeanexpensify. (Spanish can remain as it is).